### PR TITLE
New modexp cost

### PR DESCRIPTION
**Pull request by pirapira**
This follows the new cost formula https://github.com/ethereum/EIPs/commit/4d4d8fb for modexp precompiled contract.

### CODE REVIEW

**Comment by gumb0** [comment]
No unit tests change was needed - it means they cover only the case for mod & base length <=64. Could we add tests for other two cases?

File: libethcore/Precompiled.cpp [hunk 1]
```diff
@@ -160,6 +160,16 @@ namespace
 			return 8 * (_expLength - 32) + highestBit;
 		}
 	}
+
+	bigint mult_complexity(bigint const& _x)
```
Comment (gumb0) [inline libethcore/Precompiled.cpp]: I'd name it `multComplexity`

**Comment by pirapira** [comment]
Right, the unit tests are only for very small, or very large `mod | base`.

**Comment by pirapira** [comment]
I added unit tests for the other two cases.

**Comment by pirapira** [comment]
AppVeyor build is failing and somehow I cannot restart AppVeyor.

**Comment by pirapira** [comment]
The tests around `modexp` is passing in AppVeyor.

**Pull request merged**